modules/local/nacho/normalize/tests/main.nf.test

```diff
@@ -0,0 +1,35 @@
+nextflow_process {
+
+    name "Test Process NACHO_NORMALIZE"
+    script "../main.nf"
+    process "NACHO_NORMALIZE"
+
+    test("Should run without failures") {
+
+        when {
+            params {
+                pipelines_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/'
+                input  = params.pipelines_testdata_base_path + 'nanostring/samplesheets/samplesheet_test.csv'
+            }
+            process {
+                """
+                // RCC Files: Collect from samplesheet
+                input[0] = Channel.fromPath(params.input, checkIfExists: true)
+                    .splitCsv(header: true)
+                    .map { row -> row.RCC_FILE } // Select first column: path to file
+                    .collect()
+
+                // Sample sheet
+                input[1] = file(params.input, checkIfExists: true)
+                """
+            }
+        }
+
+        then {
+            assert process.success
+            assert snapshot(process.out).match()
```

Comment:
Well done on your first nf-test! To strictly adhere to nf-core guidelines, you should follow these conventions:
https://nf-co.re/docs/guidelines/components/modules#assertall

See examples here:
https://nf-co.re/docs/contributing/nf-test/assertions#nf-core-guidelines-for-assertions